khal/controllers.py

```diff
@@ -27,11 +27,12 @@
 from collections import defaultdict
 from shutil import get_terminal_size
 
-import datetime
+import datetime as dt
```

Comment:
+1 for `from datetime import datetime, date`. It makes everything below much cleaner.